Makefile

```diff
@@ -109,10 +109,14 @@ test-go: generate
 	@[ -z "$$(ls pkg/util/*.go 2>/dev/null)" ] || (echo error: go files are not allowed in pkg/util, use a subpackage; exit 1)
 	@[ -z "$$(find -name "*:*")" ] || (echo error: filenames with colons are not allowed on Windows, please rename; exit 1)
 	@sha256sum --quiet -c .sha256sum || (echo error: client library is stale, please run make client; exit 1)
+	docker run \
+		--name azurite-aro -d --rm -p 10000:10000 \
+		mcr.microsoft.com/azure-storage/azurite azurite-blob --blobHost 0.0.0.0 --blobPort 10000
```

Comment:
This is a supposition on how we can test storage account interaction.

LMK if this makes sense to use this, or if we want to mock this.

Reply:
I personally would prefer the mock so that there isn't a dependency on docker for our unit tests. But this does make sure we are properly using the go sdk for storage.